docs/light_configuration.md

```
@@ -68,6 +68,48 @@ You can also **offset** solar times:
+# Equivalent using min/max aliases
+Office  min(max(sunrise, 06:30), 08:00)  bri:254  ct:6000
+
+# Nested functions
+Hallway  notAfter(notBefore(sunrise, 06:30), 08:00)  bri:254
+
+# Combine with offsets
+Bedroom  clamp(sunrise-15, 06:30, 08:00)  bri:200  ct:3000
+```
+
+> Note: `min`/`max` accept exactly 2 arguments. For more bounds, nest calls: `max(max(a, b), c)`.
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)